lib/hexagonImage/hexagonImage.android.js

```diff
@@ -1,7 +1,9 @@
-import React, { PropTypes, Component } from 'react';
-import { View, Image, Animated, StyleSheet, Dimensions } from 'react-native';
-import { isEqual } from 'lodash';
+import { Animated, Dimensions, Image, StyleSheet, View } from 'react-native';
+import React, { Component } from 'react';
+
 import { HexagonView } from '../hexagonView';
+import PropTypes from 'PropTypes';
```

Comment:
There's on package named `PropTypes`. Probably you mean  `prop-types` .

Please test your PR before opening it in Github